### PR TITLE
First milestone service implementation

### PR DESCRIPTION
**Pull request by napalmpapalam**

### CODE REVIEW

File: internal/services/api/handlers/org_list.go [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+)
+
+type orgListRequest struct {
+	RawOwner *string `filter:"owner"`
+	Owner    *uuid.UUID
+	UserDID  *string                       `filter:"user_did"`
+	Status   *resources.OrganizationStatus `filter:"status"`
+
+	PageCursor uint64     `page:"cursor"`
+	PageLimit  uint64     `page:"limit" default:"10"`
```
Comment (ihordiachenko) [inline internal/services/api/handlers/org_list.go:26]: It's called `page_size` in every other endpoint. Is it ok?

Comment (napalmpapalam) [reply internal/services/api/handlers/org_list.go:26]: not in endpoint but in selector structs, fixed it

File: internal/notificator/notificator.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package notificator
+
+import (
+	"context"
+	"net/http"
+	"time"
+)
+
+var (
+	MailChipAPIBaseURL = "https://mandrillapp.com/api/1.0/"
```
[the remark this answers is not in the document]
Comment (napalmpapalam) [reply internal/notificator/notificator.go]: fixed

File: internal/data/groups.go [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+import (
+	"github.com/google/uuid"
+	"gitlab.com/distributed_lab/kit/pgdb"
+)
+
+type GroupsSelector struct {
+	OrgID *uuid.UUID `json:"owner,omitempty"`
+
+	PageCursor uint64     `json:"page_number,omitempty"`
```
[the remark this answers is not in the document]
Comment (napalmpapalam) [reply internal/data/groups.go]: fixed

**Review by ihordiachenko — APPROVED** [review]

**Pull request merged**